airbyte-webapp/src/hooks/services/Feature/types.tsx

```diff
@@ -14,4 +14,4 @@ export enum FeatureItem {
   AllowSyncSubOneHourCronExpressions = "ALLOW_SYNC_SUB_ONE_HOUR_CRON_EXPRESSIONS",
 }
 
-export type FeatureSet = Record<FeatureItem, boolean>;
+export type FeatureSet = Partial<Record<FeatureItem, boolean>>;
```

Comment:
It would actually have been more correct already in the past that this should be a `Partial` in the other places as well. They actually just pass us partial objects here and not the full enum. But since in the other places we usually cast it from e.g. launchdarkly this was not noticed. This type reflects the actual type we expect better.